### PR TITLE
Update sheet 6 data to 2015 sheet format.

### PR DESCRIPTION
**Pull request by esirK**
## Description

Update sheet 6 data format to match 2015 data format.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/historical/_gmmp_2015_importer.py [hunk 1]
```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
             row_heading_col=2,
         )
 
+        for year, country_data_per_media_per_region in all_data.items():
+            if year != 2010:
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]:
```suggestion
            if year > 2010:
```
One can hope in `2025` they'll be reusing our code.

File: reports/historical/_gmmp_2015_importer.py [hunk 2]
```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
...
+                # 2015 data introduced the media rows.
+                country_data_per_media_per_region = {**country_data_per_media_per_region['print, radio, television'],
+                                                        **country_data_per_media_per_region['internet, twitter']}
+            year_data = {}
+            for continent, continent_data in country_data_per_media_per_region.items():
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]:
```suggestion
            for region, region_data in country_data_per_media_per_region.items():
```
Or why the switch to `continent`?

File: reports/historical/_gmmp_2015_importer.py [hunk 3]
```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
...
+                year_data[continent] = {}
+                for _, gender in continent_data.items():
+                    data = self.combine_p_n(continent_data)
+                    year_data[continent].update(data)
```
Comment (kilemensi) [inline reports/historical/_gmmp_2015_importer.py]: Seems like we're not using anything from this loop i.e. `_` is ignored and `gender` is not used anywhere. Why do we need it?

**Review by kilemensi — COMMENTED** [review]
👍🏾 

Just to make sure I get this: from **2015** media rows were introduced and we're using that to fill `country_data_per_media_per_region`. My question is what does `country_data_per_media_per_region` contain in **2010** then?

File: reports/historical/_gmmp_2015_importer.py (hunk 3 above)
Comment (esirK) [reply reports/historical/_gmmp_2015_importer.py]: Yeah. I was testing out something. I forgot to remove this.

**Comment by esirK** [comment]
> 👍🏾
> Just to make sure I get this: from **2015** media rows were introduced and we're using that to fill `country_data_per_media_per_region`. My question is what does `country_data_per_media_per_region` contain in **2010** then?

Apologies @kilemensi  I missed this question.
In 2010, there was no separation between `print, radio, television'` and `internet, twitter` so `country_data_per_media_per_region` in 2010 contained data for all media. 
Actually, this is where I have found an issue. With the current approach, `country_data_per_media_per_region` will only have the `internet, twitter` data. So that's what I'm trying to fix.

**Pull request closed without merging**